org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/renderers/DataRenderer.java

```diff
@@ -790,7 +790,7 @@ public boolean canRenderDataType(String type) {
   }
 
   public boolean renderDataType(RenderingStatus status, XhtmlNode x, ResourceWrapper type) throws FHIRFormatError, DefinitionException, IOException {
-    return renderDataType(status, null, x, type);
+    return renderDataType(status, x, x, type);

```

Comment:
This looks a bit weird to me, as it looks like the logic adds features to both a parent node and a child, and in this case they'd be the same node.

Is there maybe a way to handle the previous null? Maybe even to use the child as a parent if that's the correct way to render.

Reply:
it is weird, but when I looked at the context, it was technically correct. Kind of screwy thing about html and <td> that do or don't contain <p>. It probably will come back as a bigger issue somewhere, but we'll let it lie for now